config/locales/admin.nl.yml

```diff
@@ -124,6 +125,20 @@ nl:
         pay_to: Kan je dit overmaken naar *NL61INGB0002877106* onder vermelding van _Openstaand %{member}_?
         title: Hey! %{user} van Sticky hier :)
         total_of: Dit geeft een totaal van *%{amount}*
+    posts:
+      author: Auteur
+      content: Bericht tekst
+      create: Bericht aanmaken
+      edit: Bericht bewerken
+      published_at: Gepubliceerd op
+      status:
+        draft: Concept
+        pinned: Gepinned
+        published: Gepubliceerd
+        review: Review
```

Comment:
What is the 'Review' status for?

Reply:
If you have created a post but you want another board member to check it, you could set it as "Review". We could also decide to ditch the review state and say that it's the same as being in draft

Reply:
I don't mind if it stays in, but I doubt people will actually use it